Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.15-alpine as builder
+
+
+RUN apk add --no-cache git make gcc musl-dev linux-headers bash
+
+WORKDIR /
+
+RUN git clone https://github.com/bisontrails/ette
```

Comment:
since we're building image within directory could do `ADD . /ette` instead of git clone

Reply:
Copy pasta from when I was building this in the ethereum repo, good catch!